app/routers/upbit_trading.py

```
@@ -0,0 +1,496 @@
+
+
+async def cancel_existing_sell_orders(market: str):
+    """해당 마켓의 기존 매도 주문들을 취소"""
+    try:
+        open_orders = await upbit.fetch_open_orders(market)
+        sell_orders = [order for order in open_orders if order.get('side') == 'ask']
+
+        if sell_orders:
+            order_uuids = [order['uuid'] for order in sell_orders]
+            await upbit.cancel_orders(order_uuids)
+    except:
```

Ruff (0.14.2)

Do not use bare `except`

(E722)